package.json

```diff
@@ -138,24 +141,35 @@
   },
   "jest": {
     "moduleNameMapper": {
-      "^\/lib(.*)": "<rootDir>/lib/$1",
-      "^\/imports\/plugins(.*)": "<rootDir>/imports/plugins/$1",
-      "^\/client\/api(.*)": "<rootDir>/imports/test-utils/__mocks__/client/api$1",
+      "^/lib(.*)": "<rootDir>/lib/$1",
+      "^/imports/plugins(.*)": "<rootDir>/imports/plugins/$1",
+      "^/client/api(.*)": "<rootDir>/imports/test-utils/__mocks__/client/api$1",
       "^meteor/aldeed:simple-schema": "<rootDir>/imports/test-utils/__mocks__/meteor/aldeed-simple-schema",
+      "^meteor/tmeasday:publish-counts": "<rootDir>/imports/test-utils/__mocks__/meteor/aldeed-simple-schema",
```

Comment:
Actually, identical to dev.. https://github.com/reactioncommerce/reaction/blob/development/package.json#L144-L149